include/megaapi.h

```diff
@@ -3491,6 +3505,38 @@ class MegaRequest
          * @return Object with information about the contents of a folder
          */
         virtual MegaBackgroundMediaUpload* getMegaBackgroundMediaUploadPtr() const;
+
+        /**
+         * @brief Returns backup id
+         *
+         * This value is valid for these requests:
+         *  - MegaApi::setBackup
+         *  - MegaApi::updateBackup
+         *  - MegaApi::sendBackupHeartbeat
+         *
+         * @return backup id
+        */
+        virtual MegaHandle getBackupId();
+
+        /**
+         * @brief Returns number of backup files uploaded
+         *
+         * This value is valid for these requests:
+         *  - MegaApi::sendBackupHeartbeat
+         *
+         * @return number of backup files uploaded
+        */
+        virtual int getBackupUploads();
+        
+        /**
+         * @brief Returns number of backup files downloaded
+         *
+         * This value is valid for these requests:
+         *  - MegaApi::sendBackupHeartbeat
+         *
+         * @return number of backup files downloaded
+        */
+        virtual int getBackupDownloads();
```

Comment:
I believe the policy is to try and avoid adding new members to MegaRequest and try to reuse existing ones (that return the same type) and document them in the corresponding getters. @sergiohs84  might have a a saying in this

Reply:
Removed the new members. Using the existing ones.